app/code/Magento/ImportExport/Controller/Adminhtml/Import/Download.php

```diff
@@ -68,22 +78,25 @@ public function __construct(
      */
     public function execute()
     {
-        $fileName = $this->getRequest()->getParam('filename') . '.csv';
-        $moduleDir = $this->componentRegistrar->getPath(ComponentRegistrar::MODULE, self::SAMPLE_FILES_MODULE);
-        $fileAbsolutePath = $moduleDir . '/Files/Sample/' . $fileName;
-        $directoryRead = $this->readFactory->create($moduleDir);
-        $filePath = $directoryRead->getRelativePath($fileAbsolutePath);
+        $entityName = $this->getRequest()->getParam('filename');
+
+        if (preg_match('/^\w+$/', $entityName) == 0) {
+            /** @var \Magento\Backend\Model\View\Result\Redirect $resultRedirect */
```

Comment:
Why do we need this hint?
` $resultRedirect` is not used in this method

Reply:
Done.